README.md

```diff
@@ -788,6 +788,23 @@ renders
 FORWARDSoneword
 ```
 
+To access a versioned secret value (for the K/V version 2 backend):
+
+```liquid
+{{ with secret "secret/passwords?version=1" }}
+{{ .Data.data.wifi }}{{ end }}
+```
+
+When omitting the `?version` parameter, the latest version of the secret will be
+fetched. Note the nested `.Data.data` syntax when referencing the secret value.
+For more information about using the K/V v2 backend, see the 
+[Vault Documentation](https://www.vaultproject.io/docs/secrets/kv/kv-v2.html).
+
+When using Vault versions 0.10.0/0.10.1, the secret path will have to be prefixed
```

Comment:
Should we also maybe be clear that V2 did not exist before 0.10.0 to limit that confusion?